src/main/resources/swagger/api-docs.yaml

```diff
@@ -5558,6 +5580,54 @@ definitions:
       - foo
       - bar
 
+  Submission:
+    description: ''
+    properties:
+      submissionId:
+        type: string
+        description: SubmissionRequest unique identifier
+      submissionDate:
+        type: string
+        format: date-time
+        description: SubmissionRequest date
+      submitter:
+        type: string
+        description: user that created the submission
+      methodConfigurationNamespace:
+        type: string
+        description: Method configuration namespace
+      methodConfigurationName:
+        type: string
+        description: Method configuration name
+      submissionEntity:
+        $ref: '#/definitions/AttributeEntityReference'
+        description: Optional if this submission is attached to a method configuration that has no root entity
+      workflows:
+        type: array
+        description: Status of Workflow(s)
+        items:
+          $ref: '#/definitions/Workflow'
+      status:
+        $ref: '#/definitions/SubmissionStatus'
+        description: Status
+      cost:
+        type: number
+        format: float
+        description: The compute cost of this submission
+      workflowFailureMode:
+        type: string
+        description: What happens after a task fails. Choose from ContinueWhilePossible and NoNewCalls. Defaults to NoNewCalls if not specified. See Cromwell docs for more information.
+        default: NoNewCalls
+        enum: ["NoNewCalls", "ContinueWhilePossible"]
+
+  SubmissionStatus:
```

Comment:
nitpick: try to keep models in alphabetical order (so this would go after `SubmissionRequest`).